### PR TITLE
Work section

### PR DESCRIPTION
**Pull request by aimalamiri**
## Work section design
1. Designed the title for the works section
2. Designed the featured projects card
3. Designed projects card
4. Designed button with all of the effects like hover and focus
5. Designed the about me section
6. Designed the technologies and skills list

Some parts of the pages are designed with utility classes, which can be useful in the future as well.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -54,5 +54,134 @@ <h2 class="header-subtitle text-white font-inter">I’m a Software Developer</h2
...
+                <li class="tag text-white dark-tag mb-6">HTML</li>
+                <li class="tag text-white dark-tag mb-6">Bootstrap</li>
+                <li class="tag text-white dark-tag mb-6">Ruby</li>
+              </ul>
+            </div>
+          </div>
+          <button class="btn w-full">See Project</button>
+        </div>
+      </section>
+    </main>
```
Comment (ShoiraTa) [inline index.html]: - [ ] You have done a truly amazing job, the best project I have seen so far! But the "About myself" section (placed after the projects and before the contact form) me section is missing, please make sure to add it into the project:
Screenshot 2022-03-17 at 20 32 37

[Link](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) to project requirements for this milestone

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @aimalamiri ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Amazing UI! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (aimalamiri) [reply index.html]: Oh, I forget that I'll fix it soon.

Comment (ShoiraTa) [reply index.html]: Good luck with the rest of the project 🤞🏻

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @aimalamiri  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍  
Congratulations! 🎉 



Cheers and Happy coding and good luck with upcoming projects👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**